### PR TITLE
merge parquet files from multiple runs

### PR DESCRIPTION
**Pull request by MikeLippincott**
Small PR to merge parquet files from multiple CP runs.

### CODE REVIEW

File: Data_Preprocessing/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Parquet File Merge
+
+This processing run was broken up into two separate runs as there was as power failure during the first half.
+
+The first and second run need to be added together
```
Comment (roshankern) [inline Data_Preprocessing/README.md:5]:
```suggestion
The first and second run still need to be added together.
```

Consider this change if it helps readability and retains original meaning.

File: Data_Preprocessing/nbconverted/file_merge.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+import numpy as np
+import pandas as pd
+import pyarrow as pa
+import pyarrow.parquet as pq
+
+# In[2]:
+
+
+# define input paths
```
Comment (roshankern) [inline Data_Preprocessing/nbconverted/file_merge.py:17]: Consider adding a comment explaining the difference between `_sc_` and `_sc_fs_` files here and why they need to be merged.

File: Data_Preprocessing/nbconverted/file_merge.py [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+first_run_sc_path = Path(
+    "/media/lippincm/c58d4f19-ae4d-4b78-8370-2c2639886da0/Interstellar_Plate2/SHSY5Y_first_run_sc_norm.parquet"
+)
+
+second_run_sc_fs_path = Path(
+    "/media/lippincm/c58d4f19-ae4d-4b78-8370-2c2639886da0/Interstellar_Plate2/SHSY5Y_second_run_sc_norm_fs.parquet"
+)
+second_run_sc_fs_path = Path(
+    "/media/lippincm/c58d4f19-ae4d-4b78-8370-2c2639886da0/Interstellar_Plate2/SHSY5Y_second_run_sc_norm.parquet"
+)
```
Comment (roshankern) [inline Data_Preprocessing/nbconverted/file_merge.py:30]: It looks like you overwrite the path here. Did you mean to do this? Probably not but if this is intentional consider explaining why with a comment.

**Review by roshankern — APPROVED** [review]
LGTM once you address the comments (especially the overwritten file name).

Thanks for the bite-sized PR :)

File: Data_Preprocessing/nbconverted/file_merge.py (hunk 3 above)
Comment (MikeLippincott) [reply Data_Preprocessing/nbconverted/file_merge.py:30]: ah, these are separate paths, the first one is _fs and the second is not. I can document which is which to add clarity

Comment (MikeLippincott) [reply Data_Preprocessing/nbconverted/file_merge.py:30]: Never mind, the path is right but not the variable name... changing now

**Pull request merged**